### DISCUSSION TITLE
Make thread responses in a PR more obvious

### DISCUSSION THREAD

**Discussion by jjspace** [community/community#8272] [Pull Requests]
If you have an existing comment thread started in a previous PR review and you submit another review where you respond to that thread you end up with a comment that has no reply box and it's unclear it's related to the full thread like below. 

![2021-11-30_12-52](https://user-images.githubusercontent.com/8007967/144101101-3d5553a5-8d2c-45e8-9cc4-7237065414bf.png)

This comment makes sense in context to the full discussion but seems random in isolation. I've had team members wonder why they can't respond to said comment right there and don't know to go back to the first thread and instead write a full comment on the PR that can't be hidden when the original comment is resolved. It wasn't until I was writing this issue up that I realized you could click the timestamp (`17 minutes ago` in this case) and it links to the main thread.

I see two possible improvements.

1. Add an item directly above as if this new comment was a thread itself with a clear link to "View full conversation"
2. Add the "Reply" input back even though you can't see the full conversation so if you know the context you're able to just reply right there without having to scroll back up to the other conversation

Additionally, somewhat unrelated so may be worth a separate discussion but comments made in this manner don't seem to disappear when you resolve the original conversation creating more lingering clutter.

**Comment by upsj** [comment]
This seems to be causing confusion even among somewhat experienced GitHub users (a few colleagues of mine, as well as the maintainer of a popular C++ unit-testing framework), and I only found out by trial-and-error. Would be great if this could be addressed, especially since the "View full conversation" solution likely requires only very minor modifications.
